eme_listeners.js

```diff
@@ -412,9 +412,10 @@ EmeListeners.logEvent = function(event) {
  * @param {string} description A short description of this Promise.
  * @param {string} status The status of this Promise.
  * @param {Object} result The result of this Promise.
+ * @param {Object} args The arguments that were passed.
```

Comment:
Should be Array as far as I can tell.  |args| comes from [].slice.call(arguments) above, which should be !Array